### ISSUE TITLE
'pagePreloaded' event doesn't pass any arguments

### ISSUE THREAD

**Issue by crankysparrow** [swup/preload-plugin#64]
The docs for this plugin include an [example for listening to the `pagePreloaded` event](https://swup.js.org/plugins/preload-plugin/#events) like this:

`swup.on('pagePreloaded', (page) => console.log('preloaded:' page));`

which simply logs `preloaded:  undefined`

The event is triggered from here in the code: https://github.com/swup/preload-plugin/blob/365df1f881fa4c6482de547e6a6fcbba6813457c/src/index.js#LL180C4-L183C32 

```
const cacheablePageData = { ...page, url };
swup.cache.cacheUrl(cacheablePageData);
swup.triggerEvent('pagePreloaded');
```

Adjusting this last line to `swup.triggerEvent('pagePreloaded', page)` or `swup.triggerEvent('pagePreloaded', url)` seems to do the trick! (I'm not sure what kind of data was initially intended to be passed to that listener) It's a small thing but would be nice to be able to grab the page info. 

I haven't made a demo or anything since I think it's pretty simple to see what's going on and replicate, but let me know if that would be helpful 😄 Thank you for the awesome library/plugin!

**Comment by hirasso** [comment]
Hey @crankysparrow ! You are right, thanks for bringing this up! I have created the PR #65 to fix this.

**Comment by hirasso** [comment]
Fixed in `2.2.1`: https://github.com/swup/preload-plugin/releases/tag/2.1.1
